lib/FrontendTool/FrontendTool.cpp

```diff
@@ -1253,17 +1253,6 @@ static bool performCompile(CompilerInstance &Instance,
   if (opts.InputsAndOutputs.shouldTreatAsLLVM())
     return compileLLVMIR(Instance);
 
-  // If we aren't in a parse-only context and expect an implicit stdlib import,
-  // load in the standard library. If we either fail to find it or encounter an
-  // error while loading it, bail early. Continuing the compilation will at best
-  // trigger a bunch of other errors due to the stdlib being missing, or at
-  // worst crash downstream as many call sites don't currently handle a missing
-  // stdlib.
```

Comment:
IMO it would be nice to preserve some of the rationale in this comment for the `loadStdlibIfNeeded` call in `setup`, what do you think?

Reply:
I’m adjusting it slightly and keeping it in.